src/pages/LogOutPreviousUserPage.tsx

```diff
@@ -70,11 +70,13 @@ function LogOutPreviousUserPage({route}: LogOutPreviousUserPageProps) {
         // because we already handle creating the optimistic policy and navigating to it in App.setUpPoliciesAndNavigate,
         // which is already called when AuthScreens mounts.
         // For HybridApp we have separate logic to handle transitions.
-        if (!NativeModules.HybridAppModule && exitTo && exitTo !== ROUTES.WORKSPACE_NEW && !isAccountLoading && !isLoggingInAsNewUser) {
+        if (!NativeModules.HybridAppModule && exitTo !== ROUTES.WORKSPACE_NEW && !isAccountLoading && !isLoggingInAsNewUser) {
```

Comment:
This is to fix the issue where this page is never popped out from the stack when exitTo is empty, showing infinite loading.